homework-1/task-3/main.js

```diff
@@ -0,0 +1,92 @@
+const bank = document.querySelector('.bank');
+const totalMoney = document.querySelector('.total');
+const moneyInput = document.querySelector('.number-money');
+const moneyBtn = document.querySelector('.money-btn');
+const render = document.querySelector('.render')
+
+let quantityMoney = [];
+
+let limits = {
+    1000: 5,
+    500: 5,
+    100: 5,
+    50: 34,
+    20: 10,
+    10: 10,
+};
+
+
+let quantity = Object.entries(limits);
+
+quantity.forEach((value) => {
```

Comment:
don't understand what are you doing here

Reply:
function of calculating the total amount of currency....maybe there is an easier way